pkg/controller/add_che.go

```diff
@@ -1,14 +1,3 @@
-//
-// Copyright (c) 2012-2019 Red Hat, Inc.
-// This program and the accompanying materials are made
-// available under the terms of the Eclipse Public License 2.0
-// which is available at https://www.eclipse.org/legal/epl-2.0/
-//
-// SPDX-License-Identifier: EPL-2.0
-//
-// Contributors:
-//   Red Hat, Inc. - initial API and implementation
-//
 package controller
```

Comment:
Sorry, what's the reason to remove licence header?

Reply:
it must have gotten removed when I rebased.  I re-added it.